config.yml

```diff
@@ -779,6 +945,265 @@ production_factories:
+        material: COAL
+        durability: 1
+    inputs:  
+      Iron Blocks:
+        material: IRON_BLOCK
+        amount: 96   
+    recipes:
+      - Create_Iron_Axe
+      - Create_Iron_Pickaxe
+      - Create_Iron_Spade
+    repair_multiple: 2
+    repair_inputs:
+      Iron:
+        material: Iron
+        amount: 9  
+  Random_Iron_Armor_Factory:
+    name: Random Iron Armor Factory
+    fuel: 
+      Charcoal:
+        material: COAL
+        durability: 1
+    inputs:
+      Iron Blocks:
+        material: IRON_BLOCK
+        amount: 64   
+    recipes:
+      - Create_Iron_Leggings
+      - Create_Iron_Boots
+      - Create_Iron_Chestplate
+      - Create_Iron_Helmet
+    repair_multiple: 2
+    repair_inputs:
+      Iron Blocks:
+        material: IRON_BLOCK
+        amount: 6  
+  Random_Iron_Weapon_Factory:
+    name: Random Iron Weapon Factory
+    fuel: 
+      Charcoal:
+        material: COAL
+        durability: 1
+    inputs:
+      Iron Blocks:
+        material: IRON_BLOCK
+        amount: 32  
+    recipes:
+      - Create_Iron_Sword
+    repair_multiple: 2
+    repair_inputs:
+      Iron Blocks:
+        material: IRON_BLOCK
+        amount: 3  
+  Random_Diamond_Tool_Factory:
+    name: Random Diamond Tool Factory
+    fuel: 
+      Charcoal:
+        material: COAL
+        durability: 1
+    inputs:  
+      Emerald Blocks:
+        material: EMERALD_BLOCK
+        amount: 96   
+    recipes:
+      - Create_Diamond_Axe
+      - Create_Diamond_Pickaxe
+      - Create_Diamond_Spade
+    repair_multiple: 2
+    repair_inputs:
+      Emerald Blocks:
+        material: EMERALD_BLOCK
+        amount: 9  
+  Random_Diamond_Armor_Factory:
+    name: Random Diamond Armor Factory
+    fuel: 
+      Charcoal:
+        material: COAL
+        durability: 1
+    inputs:
+      Emerald Blocks:
+        material: EMERALD_BLOCK
+        amount: 64   
+    recipes:
+      - Create_Diamond_Leggings
+      - Create_Diamond_Boots
+      - Create_Diamond_Chestplate
+      - Create_Diamond_Helmet
+    repair_multiple: 2
+    repair_inputs:
+      Emerald Blocks:
+        material: EMERALD_BLOCK
+        amount: 6 
+  Random_Diamond_Weapon_Factory:
+    name: Random Diamond Weapon Factory
+    fuel: 
+      Charcoal:
+        material: COAL
+        durability: 1
+    inputs:
+      Emerald Blocks:
+        material: EMERALD_BLOCK
+        amount: 32  
+    recipes:
+      - Create_Bow
+      - Create_Diamond_Sword
+    repair_multiple: 2
+    repair_inputs:
+      Emerald Blocks:
+        material: EMERALD_BLOCK
+        amount: 3  
+  Sword_Enchantment_Factory:
+    name: Sword Enchantment Factory
+    fuel:
+      Charcoal:
+        material: COAL
+        durability: 1
+    inputs:
+      Diamond Block:
+        material: DIAMOND_BLOCK
+        amount: 32
+      Emerald Block:
+        material: EMERALD_BLOCK
+        amount: 64
+    recipes:
+      - Enchant_Sharpness
+      - Enchant_Fire_Aspect
+      - Enchant_Looting

```

Comment:
Bane of Anthropods is p. important for animal enthusiasts, I'd recommend it be added here as an achievable recipe.